OIPA/api/activity/tests/test_save_serializers.py

```diff
@@ -2386,6 +2389,7 @@ def test_update_transaction(self):
         self.assertEqual(instance.transaction_type.code, data['transaction_type']['code'])
         self.assertEqual(instance.disbursement_channel.code, data['disbursement_channel']['code'])
         self.assertEqual(instance.humanitarian, data['humanitarian'])
+        self.assertEqual(instance.transactionrecipientcountry_set.all()[0].country.code, data['recipient_country']['country']['code'])
```

Comment:
Can you just add a check that the reported_transaction fk is set correctly as well? So check with the related_name you have added on the model